.markdownlint.yaml

```diff
@@ -0,0 +1,229 @@
+# MD001/heading-increment/header-increment - Heading levels should only increment by one level at a time
+MD001: true
+
```

Comment:
Why?

Reply:
Link at top is for convenience since it contains a lot more information about each rule.

Defaulting to `false` makes it so we don't accidentally get a new rule without it being reviewed/agreed on by the various members (e.g., if we update the action to a newer version that includes a new check, it shouldn't default to enabled).